File: Planarian/Planarian/Modules/Caves/Services/CaveCsvModel.cs
namespace Planarian.Modules.Caves.Services;

public class CaveCsvModel   
{
    public string CaveName { get; set; }
    public double CaveLengthFt { get; set; }
    public double CaveDepthFt { get; set; }
    public double MaxPitDepthFt { get; set; }
    public int NumberOfPits { get; set; }
    public string? Narrative { get; set; }
    public string CountyCode { get; set; }
    public string CountyName { get; set; }
    public int CountyCaveNumber { get; set; }
    public string State { get; set; }
    public string? Geology { get; set; }
    public string? ReportedOnDate { get; set; }
    public string? ReportedByName { get; set; }
    public bool? IsArchived { get; set; }
}

File: Planarian.Web/src/Modules/Import/Models/CaveCsvModel.ts
export interface CaveCsvModel {
  caveName: string;
  caveLengthFt: number;
  caveDepthFt: number;
  maxPitDepthFt: number;
  numberOfPits: number;
  narrative: string;
  countyCode: string;
  countyName: string;
  countyCaveNumber: number;
  state: string;
  geology: string;
  reportedOnDate: string;
  reportedByName: string;
  isArchived: boolean;
}
